packages/extensionmanager/src/model.ts

```diff
@@ -343,9 +356,13 @@ export class ListModel extends VDomModel {
    * Install an extension.
    *
    * @param entry An entry indicating which extension to install.
+   * @param options Additional options for the action.
    */
-  async install(entry: IEntry): Promise<void> {
-    await this.performAction('install', entry).then(data => {
+  async install(
+    entry: IEntry,
+    options: { useVersion?: string } = {}
```

Comment:
@fcollonval - Thanks for the review! I also gave a default value to this parameter here, but let me know if you'd rather it not be optional.